jme3-core/src/main/java/com/jme3/util/NativeObject.java

```diff
@@ -249,4 +259,17 @@ public <T> WeakReference<T> getWeakRef() {
         }
         return (WeakReference<T>) weakRef;
     }
+
+    public boolean isValid() {
+        return valid;
+    }
+
+    public boolean isInvalid() {
```

Comment:
I propose removing this method and using !isValid() whenever needed. 
I think it's generally good practice to avoid "negative" method names.
WDYT?

Reply:
Yes, you are right, and it's redundant anyway.